generators/server/templates/_pom.xml

```diff
@@ -1037,25 +1040,48 @@
             <build>
                 <plugins><% if(!skipClient) {%>
                     <plugin>
-                        <groupId>com.github.trecloux</groupId>
-                        <artifactId>yeoman-maven-plugin</artifactId>
-                        <version>0.5</version>
+                        <groupId>com.github.eirslett</groupId>
+                        <artifactId>frontend-maven-plugin</artifactId>
+                        <version>0.0.27</version>
                         <executions>
                             <execution>
-                                <id>run-frontend-build</id>
-                                <phase>generate-resources</phase>
+                                <id>install node and npm</id>
                                 <goals>
-                                    <goal>build</goal>
+                                    <goal>install-node-and-npm</goal>
                                 </goals>
                                 <configuration>
-                                    <buildTool>gulp</buildTool>
-                                    <buildArgs>build --no-notification</buildArgs>
+                                    <nodeVersion>v5.10.1</nodeVersion>
```

Comment:
Configuration should use Node LTS as we now recommend in http://jhipster.github.io/installation/